modules/modal/react/lib/Modal.tsx

```diff
@@ -27,12 +27,7 @@ const Modal = ({
 }: ModalProps): JSX.Element | null =>
   // Only render if on the client and `open` is `true`
   open && typeof window !== 'undefined' ? (
-    <ModalContent
-      container={container || document.body}
-      padding={padding}
-      width={width}
-      {...modalContentProps}
-    />
+    <ModalContent container={container} padding={padding} width={width} {...modalContentProps} />
```

Comment:
We can no longer set the default of the container to `document.body` here. It will short-circuit `ModalContent`. We'll need to look into deprecating `container` entirely. Setting `container` will probably seriously mess things up.